CHANGELOG.md

```diff
@@ -1,3 +1,13 @@
+## 1.0.0
+
+### Breaking Changes
+
+None
```

Comment:
Shouldn't removal of node 14 support be here?

Reply:
Oops. Yes it should have.

Reply:
#106